app/src/theme/overrides/react_tooltip_styles_override.tsx

```diff
@@ -73,7 +73,7 @@ export const ReactTooltipStylesOverride = css`
   }
 
   .scalarValueTooltip {
-    ${DefaultTooltip}
+    ${DefaultTooltip};
     padding: 5px 8px !important;
     font-weight: 500 !important;
```

Comment:
and these

Reply:
Only green code is written to this pr here I just put ";" cause it was screaming some small error at me . Dunno how else would this be written though?